tremor-script/src/script.rs

```diff
@@ -58,6 +59,8 @@ pub enum Return<'event> {
 pub struct Script {
     /// Rental for the runnable script
     pub script: crate::ast::Script<'static>,
+    /// Rental for the runnable script
+    pub named: HashMap<String, crate::ast::Script<'static>>,
```

Comment:
could we consolidate all the scripts into this map? the `script` above is also just listening on the `in` port, isn't it? And a `script from in` would overwrite this script, no?

Reply:
we coud but I don't think we should. the `in` port is the 90% case and avoiding a hashmap lookup is nice for that.

Reply:
True dat